client/src/TagCopy.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TagCopy.css';
+
+export default function Tag(props) {
+  const edit = props.edit;
+  const [existance, setExistance] = React.useState(true);
+
+  function renderDisplay() {
+    return (
+      <div className="tagShape" edit={props.edit}>
+        {props.text}
+      </div>
+    );
+  }
+
+  function renderEdit() {
+    return (
+      <div className="tagShapeEdit" edit={props.edit}>
+        <button
+          className="tagDeleteButton"
+          onClick={() => props.onDelete(props.index)}
+        >
+          ⓧ
```

Comment:
let's have the x button on the right side of the text to make it consistent with the filtering tags too :)